jobbergate-docs/src/developer_guide/local_development.rst

```diff
@@ -0,0 +1,269 @@
+=========================
+ Local Development Setup
+=========================
+
+There are several options available for running parts of Jobbergate locally. This
+document will walk through the most important ones.
+
+
+Local jobbergate-cli setup
+--------------------------
+
+The ``jobbergate-cli`` is the easiest component to set up locally. It is designed so
+that it may be deployed anywhere and only needs to be connected to an API.
+
+
+Create a new application in Auth0
+.................................
+
+Before you begin, you will need to have an Auth0 application setup for your instance of
+the ``jobbergate-cli``. Request one of the senior Armada engineers to set one up for
+you. When the setup is done, you will be provided two values that you will need for
+configuration:
+
+* ``client_id``
+* ``client_secret``
+
+Save these for later.
+
+
+Clone the git repo
+..................
+
+Start by cloning the `jobbergate <https://github.com/omnivector-solutions/jobbergate>`_
+repository and changing to the ``jobbergate-cli`` directory:
+
+.. code-block:: bash
+
+   $ git clone git@github.com:omnivector-solutions/jobbergate.git
+   $ cd jobbergate/jobbergate-cli
+
+
+Install the python package
+..........................
+
+Next, you will need to install the package. The ``jobbergate-cli`` package uses
+`Poetry <https://pytyhon-poetry.org/>`_ for dependency and virtualenv management. Run
+this command to install the package:
+
+.. code-block:: bash
+
+   $ poetry install
+
+
+Configure things
+................
+
+Now, you will need to set up a local ``.env`` file containing some required settings.
+You will need to use the ``client_id`` and ``client_secret`` you got from the Auth0 setup
+step.
+
+Below is the minimum set of configuration values you will need to set::
+
+   JOBBERGATE_API_ENDPOINT=https://armada-k8s.staging.omnivector.solutions/jobbergate
+   AUTH0_DOMAIN=omnivector.us.auth0.com
+   AUTH0_LOGIN_DOMAIN=login.omnivector.solutions
+   AUTH0_AUDIENCE=https://armada.omnivector.solutions
+   AUTH0_CLIENT_ID=<client_id>
+   AUTH0_CLIENT_SECRET=<client_secret>
```

Comment:
Outdated, some names here changed with the migration to keycloak

Reply:
Damn, they sure are!